core/src/main/java/hudson/model/User.java

```diff
@@ -708,7 +709,7 @@ private boolean relatedTo(@NonNull AbstractBuild<?, ?> b) {
     @WithBridgeMethods(List.class)
     public @NonNull RunList getBuilds() {
         return RunList.fromJobs((Iterable) Jenkins.get().
-                allItems(Job.class)).filter((Predicate<Run<?, ?>>) r -> r instanceof AbstractBuild && relatedTo((AbstractBuild<?, ?>) r));
+                allItems(Job.class)).filter((Predicate<Run<?, ?>>) this::relatedTo);
     }
```

Comment:
Code is all fine and dandy, bit the javadoc comment was wrong previously (this includes SCM change and triggers by the user). Would make a suggestion but outside of the code diff.